build/Test.targets

```diff
@@ -16,12 +11,11 @@
       <PathListSeparator Condition=" '$(OSName)' == 'win' ">%3b</PathListSeparator>
       <ExecPath>$(Stage2Directory)$(PathListSeparator)$(PATH)</ExecPath>
       <ExecPath Condition=" '$(OSName)' == 'win' ">$(Stage2Directory)</ExecPath>
-      <TestArtifactsEnv>$(TestArtifactsDir)</TestArtifactsEnv>
     </PropertyGroup>
```

Comment:
I felt that these properties could remain here since they depend on environment variables. Or should they also be separated out into a static props file?